packages/celotool/src/cmds/bots/auto-vote.ts

```diff
@@ -302,10 +302,7 @@ function shouldChangeVote(
   scoreSensitivity: BigNumber,
   baseChangeProbability: BigNumber
 ): boolean {
-  const scoreBasedProbability = score
-    .pow(scoreSensitivity)
-    .negated()
-    .plus(1)
+  const scoreBasedProbability = score.pow(scoreSensitivity).negated().plus(1)
```

Comment:
I personally prefer the old version here, interested to hear what others think.

Is this change due to a larger line width now allowed by V2?

Reply:
I believe it's a new heuristic they're using to chain in some circumstances and keep stuff on one line in other cases ([first section here](https://prettier.io/blog/2020/03/21/2.0.0.html#javascript))

Reply:
I see I see, looks like there's a lot of work has gone into making this feature land. Doesn't look like trying to configure this away will work.

Reply:
I prefer the new style, the old style seemed unnecessarily narrow. I'm generally in favour of keeping stuff on one line, as long as its easily readable and the line is not so long that it's likely it will get wrapped for someone viewing the code.